pages/about.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Architectural Summit | About</title>
+    <link rel="stylesheet" href="../css/style.css" />
+  </head>
+  <body>
+    <header>
+      <ul class="social">
+        <li>
+          <a href="http://web.facebook.com/lawrence.amoafo"
+            ><ion-icon name="logo-facebook" id="icon"></ion-icon
+          ></a>
+        </li>
+        <li>
+          <a href="http://twitter.com/lawrence_amoafo"
+            ><ion-icon name="logo-twitter" id="icon"></ion-icon
+          ></a>
+        </li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+
+      <nav class="head">
+        <div class="vector">
+          <span></span>
+          <span></span>
+          <span></span>
+        </div>
+
+        <div class="logo">
+          <a href="../index.html"
+            ><img src="../Images/logo.png" alt="logo"
+          /></a>
+          <a href="../index.html"><b>World Architectural Summit 2023</b></a>
+        </div>
+
+        <ul class="nav-links">
+          <li><a href="../index.html">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#location">Location</a></li>
+          <li><a href="#archives">Archives</a></li>
+        </ul>
+
+        <ul class="nav-links2">
+          <li><a href="../index.html">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#location">Location</a></li>
+          <li><a href="#archives">Archives</a></li>
+          <li class="style">WA Summit</li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
```

Comment:
- [ ] In the About page, the original design's sections have borders that divide them, but I cannot see sections having borders in your design so kindly consider fixing it.

Original design
![po](https://user-images.githubusercontent.com/96683843/226001497-f9e61961-9e53-4c9b-be6c-d7976edb6049.jpg)